frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -490,6 +490,7 @@ function MathSelector({ math, mathGroupTypeIndex, index, onMathSelect, style }:
             data-attr={`math-selector-${index}`}
             dropdownMatchSelectWidth={false}
             dropdownStyle={{ maxWidth: 320 }}
+            listHeight={280}
```

Comment:
Should this be a max-height in case we have variable-length lists? Are we going to do anything to show a clipped item to encourage users to scroll?

Reply:
This is just one list with a fixed set of items that got the treatment. With this PR I wanted to solve the immediate pain points without overthinking it.